src/System.Management.Automation/engine/hostifaces/ChoiceDescription.cs

```diff
@@ -19,6 +19,10 @@ namespace System.Management.Automation.Host
     public sealed
     class ChoiceDescription
     {
+        // DO NOT REMOVE OR RENAME THESE FIELDS - it will break remoting
```

Comment:
I think the warning messages should say "it will break remoting compatibility with Windows PowerShell".

Reply:
This is incomplete.  To fully fix this incompatibility we need to revert field names for all encoded class/structs.  Here is the supported list:
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs#L41

Reply:
I reused an existing comment.  I can update all occurrences of that comment to be more specific.  Thanks for the pointer to the list of types, I'll fix those as well.